src/Blocks/QueryLoopExtension.php

```diff
@@ -43,6 +43,35 @@ public static function registerFrontendQuery(): void
             'query_loop_block_query_vars',
             function ($query, $block) {
                 $blockQuery = $block->context['query'] ?? [];
+
+                // This applies only to Actions List block
+                if ($blockQuery['isCustom']) {
+                    $query['post_type'] = ['page'];
+                    $is_new_ia = !empty(planet4_get_option('new_ia'));
+
+                    if (!$is_new_ia) {
+                        $query['post_parent'] = planet4_get_option('take_action_page');
+                    } else {
+                        $query['post_type'][] = 'p4_action';
+                        $query['post_parent__in'] = [planet4_get_option('take_action_page')];
+                        $query['posts_per_page'] = -1;
+                    }
+
+                    if (is_array($blockQuery['orderBy'])) {
```

Comment:
Take [this](https://github.com/greenpeace/planet4-master-theme/pull/2492/files#diff-ad9c111941a060027e21059b76d6136f4b1edb3aa4744e8f094f82ad200e3260R33-R34) or set by default https://github.com/greenpeace/planet4-master-theme/pull/2492/files#diff-5731321646bfd7fe6c8f5582c07ece1a8c4078579f40d518eafc522c4f89f421R66-R70